### DISCUSSION TITLE
Welcome to wud Discussions!

### DISCUSSION THREAD

**Discussion by fmartinou** [getwud/wud#461] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kenneth-tarning** [comment]
Hello, I’ve been running wud on a couple of docker nodes and it's fabulous. I'm curious if there is/are going to be support for other providers like Swarm?

**Reply by fmartinou** [reply]
Hi, Swarm is not yet supported but it might be added if there is some interest from the community; I've created a dedicated thread for it there https://github.com/getwud/wud/discussions/472.
